### DISCUSSION TITLE
Stopping sites being added to Recently Visited

### DISCUSSION THREAD

**Discussion by gutley** [mozilla-mobile/firefox-ios#13532] [Q&A] [answered]
This is a related to https://github.com/mozilla-mobile/firefox-ios/discussions/13522.

Once my fork has blocked a page from being shown, I don't want it to show up in the Recently Visited section of the home page:

<img width="633" alt="image" src="https://user-images.githubusercontent.com/2040690/224315933-aacf4eb4-eb68-4b51-a89e-826a882666d7.png">

Can someone point me to where pages get added to this list so I can stop it at source.

**Comment by lmarceau** [comment, marked as answer]
Heya, you can do as it was done with `SponsoredContentFilterUtility` to block certain pages to show in Recently Visited. You can have something similar as `filterSponsoredHighlights`.
